README.md

```diff
@@ -4,14 +4,14 @@ A super minimal implementation of the Solaris backend.
 
 ## Usage
 
-### Build and run
+### Running with hot reloading enabled
 
-```
-docker build -t mocksolaris .
-docker run -p 127.0.0.1:2091:2091 mocksolaris
-```
+You need two terminal sessions:
 
-You may now access the backoffice at [http://localhost:2091/\_\_BACKOFFICE\_\_/](http://localhost:2091/__BACKOFFICE__/).
+1. `npm run build:docker`
```

Comment:
We still nneed to run the `npm build` before, right?

Reply:
This (4385f78) would be the proper way of running things. Thanks for trying it out!

Reply:
I did a clean install and all runs smooth, although I haven't been able to test together with the BE (left a comment over there already).

Reply:
Replied there as well. Don't mix the two things, they are separate.

You should either start mock-solaris as described in the README of this repo, or, by following the instructions in the backend repository. One or the other. Don't mix them 😄